pkg/build/controller/build/build_controller.go

```diff
@@ -36,20 +34,23 @@ import (
 	configv1 "github.com/openshift/api/config/v1"
 	imagev1 "github.com/openshift/api/image/v1"
 	buildv1client "github.com/openshift/client-go/build/clientset/versioned"
+	buildclientv1 "github.com/openshift/client-go/build/clientset/versioned/typed/build/v1"
```

Comment:
This is bad, we're importing 2 similar stuff, I'd prefer we merge usage of the two.